jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/ProjectLabelEnricher.java

```diff
@@ -54,7 +54,8 @@
 
     @AllArgsConstructor
     private enum Config implements Configs.Config {
-        USE_PROJECT_LABEL("useProjectLabel", "false");
+        USE_PROJECT_LABEL("useProjectLabel", "false"),
+    	CUSTOM_APP_NAME("customAppName", "");
```

Comment:
I think leaving the config value just as:
```suggestion
    	APP("app", "");
```
would make more sense and is probably aligned with most of the other enrichers (configuration key matched target property name)

`jkube.enricher.jkube-project-label.app` is also clearer than `jkube.enricher.jkube-project-label.customAppName`